src/software/guides/using-in-dotnet.md

```diff
@@ -10,6 +10,8 @@ The .NET SDK tools provide an easy way to integrate MyGeotab into .NET software.
 
 The [nuget package](https://www.nuget.org/packages/Geotab.Checkmate.ObjectModel/) is an SDK library for accessing MyGeotab customer databases. It is a convenient "wrapper" around Geotab’s HTTP/JSON API to allow developers focus on writing code instead of moving data over the wire. It includes tools to assist authenticating against Geotab’s servers, automatically serializing/deserializing JSON, and providing definitions for Checkmate object classes.
 
+Please remember to regularly check for updates to the Checkmate DLL. Ideally, your integration should use the same Checkmate DLL version as the one that your database is on. 
```

Comment:
I don't think we should refer to this as the "Checkmate DLL" rather "Geotab.Checkmate.ObjectModel nuget package"